packages/connectivity/src/scp-cf/environment-accessor/service-credentials.ts

```diff
@@ -43,13 +43,12 @@ export function getServiceCredentials<
     if (credentials) {
       return credentials;
     }
-
     logger.debug(
-      'Could not match binding service instance using JWT client id or audience.'
+      "Could not match the service instance to the JWT's client id or audience."
     );
-  } else {
-    logger.debug(`No JWT given to select binding to service '${service}.`);
+    return undefined;
```

Comment:
[req] Instead of this you could write this code a bit more readable:

```ts

if (!credentials) {
    // log debug message
}

return credentials
```